src/scene/vcPOI.cpp

```diff
@@ -873,19 +876,17 @@ void vcPOI::HandleImGui(vcState *pProgramState, size_t *pItemID)
     vdkProjectNode_SetMetadataString(m_pNode, "textSize", pTemp);
   }
 
-  // Handle hyperlinks
-  const char *pHyperlink = m_metadata.Get("hyperlink").AsString();
-  if (pHyperlink != nullptr)
+  if (vcIGSW_InputText(vcString::Get("scenePOILabelHyperlink"), m_hyperlink, vcMaxPathLength, ImGuiInputTextFlags_EnterReturnsTrue))
+    vdkProjectNode_SetMetadataString(m_pNode, "hyperlink", m_hyperlink);
+
+  if (m_hyperlink[0] != '\0' && ImGui::Button(vcString::Get("scenePOILabelOpenHyperlink")))
   {
-    ImGui::TextWrapped("%s: %s", vcString::Get("scenePOILabelHyperlink"), pHyperlink);
-    if (udStrEndsWithi(pHyperlink, ".png") || udStrEndsWithi(pHyperlink, ".jpg"))
-    {
-      ImGui::SameLine();
-      if (ImGui::Button(vcString::Get("scenePOILabelOpenHyperlink")))
-        pProgramState->pLoadImage = udStrdup(pHyperlink);
-    }
+    if (udStrEndsWithi(m_hyperlink, ".png") || udStrEndsWithi(m_hyperlink, ".jpg"))
+      pProgramState->pLoadImage = udStrdup(m_hyperlink);
+    else
+      vcWebFile_OpenBrowser(m_hyperlink);
   }
-
+  
```

Comment:
If you don't already have the VS extension installed, you need to install EditorConfig - this should sort out these whitespace issues you've had in this PR.

Reply:
fxied i installed codemaid